jaxrs-clients/src/test/java/com/palantir/remoting3/jaxrs/JaxRsClientFailoverTest.java

```diff
@@ -118,10 +118,23 @@ public void testConnectionError_performsFailoverOnDnsFailure() throws Exception
 
     @Test
     public void testQosError_performsFailover() throws Exception {
-        server1.enqueue(new MockResponse().setResponseCode(503));
+        server1.enqueue(new MockResponse().setResponseCode(UNAVAILABLE_RESPONSE_CODE));
```

Comment:
think I prefer the 503 here

Reply:
can you add a similar test for okhttp-clients?

Reply:
sure